File: 프로그래머스/unrated/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.java
class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        //swap문제
        for(int[] query : queries){
            int x = query[0], y = query[1];
            int temp = arr[x];
            arr[x] = arr[y];
            arr[y] = temp;
        }
        return arr;
    }
}

File: 프로그래머스/0/181924. 수열과 구간 쿼리 3/수열과 구간 쿼리 3.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    for(vector<int> q : queries){
        int p1 = q[0];
        int p2 = q[1];
            swap(arr[p1], arr[p2]);
    }
    return arr;
}